src/ontology/metadata/doid.yml

```diff
@@ -11,4 +11,5 @@ description: >
   (https://disease-ontology.org/).
 homepage: https://disease-ontology.org/
 base_prefix_map:
-  DO: http://purl.obolibrary.org/obo/DOID_
\ No newline at end of file
+  DO: http://purl.obolibrary.org/obo/DOID_
+  DOID: http://purl.obolibrary.org/obo/DOID_
```

Comment:
@matentzn I'm not sure if `DO` was an error on your part. In the normal `prefix_map` part of this file, it's `DOID`.

Needed to add `DOID` to this in order for the code to work.

Reply:
Yeah, my mistake. SHould always be `DOID`

Reply:
Ok np. I'll remove the `DO`

Reply:
thx